applications/crossbar/doc/phone_numbers.md

```diff
@@ -220,12 +220,30 @@ Possible reasons for failure:
 
 > POST /v2/accounts/{ACCOUNT_ID}/phone_numbers/{PHONENUMBER}
 
-```curl
+```shell
 curl -v -X POST \
     -H "X-Auth-Token: {AUTH_TOKEN}" \
+    -d '{"data": {"my_field": "some other value"}}' \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/phone_numbers/{PHONENUMBER}
 ```
 
+```json
+{
+    "auth_token": "694c7e1938aa213955fe600442c433f1",
```

Comment:
use the placeholders `{AUTH_TOKEN}`, `{ACCOUNT_ID}`, etc. Sadly, there are people that will complain if their response doesn't match this one unless we use the placeholders.